src/main/java/homework_2/TrafficLight.java

```diff
@@ -0,0 +1,73 @@
+package homework_2;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.time.LocalTime;
+import java.time.format.DateTimeParseException;
+
+public class TrafficLight {
+
+    private static boolean isDigits(String str) {
+        try {
+            Integer.parseInt(str);
+            return true;
+        } catch (NumberFormatException e) {
+            return false;
+        }
+    }
+
+    private static boolean isLocalTime(String str) {
+        try {
+            LocalTime.parse(str);
```

Comment:
Opt: if null is passes, then exception is not handled

Reply:
I added handling NPE but if I use CTRL + D hotkeys I get an infinite loop. Should I just stop the program if a null is passed?

Reply:
I decided to stop the program. Because if we pass a null from a keyboard, it means we close the input stream and we will not able to pass something else since that moment, isn't it?